lib/pages/faq/SearchBar.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:acs_upb_mobile/widgets/selectable.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class SearchBar extends StatefulWidget {
```

Comment:
Move this file to `widgets/`, it is generic and reusable. Call it `search_bar.dart`.